pageserver/src/bin/pageserver.rs

```diff
@@ -77,6 +78,9 @@ fn main() -> anyhow::Result<()> {
         )
     })?;
 
+    let sentry_url = arg_matches.get_one::<String>("sentry-url");
+    let _guard = init_sentry(sentry_url, "pageserver");
```

Comment:
`_sentry_guard` here and in other places?

Reply:
I don't think `_sentry_` prefix adds value here. if `_guard` is returned from `init_sentry`, it obviously has to do with sentry.

Reply:
It is, but I think it's better if a variable's name is descriptive without reading its initialization. Not a big deal, your call.

Reply:
I opted for _sentry_guard since I agree with yeptons's reasoning.